### DISCUSSION TITLE
не вывод новый пост

### DISCUSSION THREAD

**Discussion by jorjik** [maxsite/cms#448] [Q&A]
сайт после переноса; в админке всё норм https://i.vgy.me/We0yJS.png , но на главной и по УРЛ не выводится http://2donow.site/blog/page/kak-uskorit-zagruzku-geo-karty

**Comment by maxsite** [comment]
Может время на сервере ещё не наступило. Проверьте временную поправку в админке или попробуйте время записи поставить на день раньше. Будет понятно.

**Reply by jorjik** [reply]
помогло!
